docs/tutorials/google/colab.ipynb

```diff
@@ -33,7 +33,7 @@
         "## How to download iPython notebooks from github\n",
         "\n",
         "You can retrieve ipython notebooks in the cirq repository by\n",
-        "going to the [doc directory](https://github.com/quantumlib/Cirq/tree/master/docs).  Select the file that you would like to download and then click the \"Raw\" button in the upper right part of the window:\n",
+        "going to the [doc directory](https://github.com/quantumlib/Cirq/tree/master/docs).  For instance, this colab template can be found [here](https://github.com/quantumlib/Cirq/blob/master/docs/tutorials/google/colab.ipynb).  Select the file that you would like to download and then click the \"Raw\" button in the upper right part of the window:\n",
```

Comment:
You could just add the raw link as well?

Reply:
Done.  For some reason, I thought the raw link would pin it to a specific version, but it looks like that's not the case.